### ISSUE TITLE
Can't Reproduce Result in CIFAR-100 with 10% label

### ISSUE THREAD

**Issue by lllllljb** [nayeemrizve/TRSSL#5]
Hi,

Thanks for the great work. I download the code and directly run it several times without modifying. 'CUDA_VISIBLE_DEVICES=2,3 python3 train.py --dataset cifar100 --lbl-percent 10 --novel-percent 50 --arch resnet18 '. 
For seen class, i can get the ACC 68%  , however, for the novel class i can't get a result better than 49% while the paper had 52.1%. And even with random seed, i can't reproduce Model Training Results.  And the final results are not so stable .  Did i miss something?

Thanks!

**Comment by nayeemrizve** [comment]
Hello,

Thank you for expressing interest in our work. The command seems fine. It might be because of some bug introduced while refactoring the code for this repo or can be some version mismatch. I am looking into this. Will get back to you after running some tests. I have added some sample logs for the CIFAR-100 training that we used for reporting the scores. 
[log-cifar100-run1.txt](https://github.com/nayeemrizve/TRSSL/files/10899391/log-cifar100-run1.txt)
[log-cifar100-run2.txt](https://github.com/nayeemrizve/TRSSL/files/10899394/log-cifar100-run2.txt)

**Comment by GritCS** [comment]
Thanks for auchor‘s  reply. But I wonder when you share with us  a corrected code. Thanks for the great work.

**Comment by lllllljb** [comment]
Hi,
Thank you for your reply. Here is my training log. I found loss in my log is smaller and the random seed do not seem to be working.
[log.txt](https://github.com/nayeemrizve/TRSSL/files/10940178/log.txt)

Any updata?Could you please share requirement.txt?

**Comment by nayeemrizve** [comment]
Thank you for being so patient. I have updated the code. I have attached a training log with the updated code. I used pytorch 1.7.1 and torchvision 0.8.2 for this run.
[newlog_cifar100.txt](https://github.com/nayeemrizve/TRSSL/files/10973655/newlog_cifar100.txt)
